site/components/case-study-overview/index.js

```diff
@@ -24,7 +24,10 @@ import arrowSVG from '../../../assets/images/SVG/arrow.svg';
 
 const CaseStudyOverview = () => (
   <section className={styles.caseStudyContainer}>
-    <h2 className={styles.heading}>We solve complex problems and deliver real impact.</h2>
+    <h2 className={styles.heading}>
+      Our proven track record of being catalysts for change for our clients, positively impacting
+      their culture and leaving a lasting legacy of a process and great internal capability.
```

Comment:
are the line breaks here and later on going to cause problems at smaller screen widths? I guess this would come out in a design review